tests/utils.py

```diff
@@ -42,6 +42,10 @@
 POSTGRES_PASSWORD = os.environ.get("SYNAPSE_POSTGRES_PASSWORD", None)
 POSTGRES_BASE_DB = "_synapse_unit_tests_base_%s" % (os.getpid(),)
 
+# When debugging a specific test, it's occasionally useful to write the
+# DB to disk and query it with the sqlite CLI.
+SQLITE_PERSIST_DB = os.environ.get("SYNAPSE_TEST_PERSIST_SQLITE_DB") is not None
```

Comment:
Why `str(...)` --- are the values bytes by default?

Come to think of it, maybe I could do `"VARNAME" in os.environ`, since I'm only interested in if it's set or not. I'll probably leave it be for now though.

Reply:
> Why `str(...)` --- are the values bytes by default?

It makes `None` -> `"None"` 😄 

This is only really useful if you want to ensure that `true`-ish string values are used.